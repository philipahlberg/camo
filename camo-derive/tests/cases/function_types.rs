use camo_derive::Camo;

#[derive(Camo)]
struct Foo {
    f: fn(i32) -> i32,
}

fn main() {}
