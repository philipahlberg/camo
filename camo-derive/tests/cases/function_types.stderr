error: `camo` does not support function types
 --> tests/cases/function_types.rs:5:8
  |
5 |     f: fn(i32) -> i32,
  |        ^^
